tests/test_io.py

```diff
@@ -91,6 +92,19 @@ def test_load_population():
     npt.assert_array_equal(names, names1)
     assert population.neurons[0].is_equal(population1.neurons[0])
 
+    population2 = io.load_population(os.path.join(POP_PATH, 'sample.swc'))
+    assert len(population2.neurons) == 1
+
+    names2 = np.array([os.path.basename(n.name) for n in population2.neurons])
+    npt.assert_array_equal(['sample'], names2)
+    assert population.neurons[0].is_equal(population2.neurons[0])
+
+    with pytest.raises(
+        TypeError,
+        match='The type of the given neurons is not supported'
+    ) as excinfo:
+        io.load_population(os.path.join(POP_PATH, 'UNKONW'))
+
```

Comment:
For completeness could you also please add a case using `pathlib.Path` objects?

Reply:
My suggestion was to ensure that pathlib paths work too. Why does it fail?

I would expect that it doesn't fail because it is a pathlib path, rather because it is an unknown path. os.path methods should work with pathlib objects too I believe.

Reply:
Ah ok, sorry.
`pathlib.Path` objects do not work with the master branch but I fixed this in the last commit.

Reply:
ic, I missed the part where it is assumed they are strings further below. Sorry for the confusion.